common/src/main/resources/runtime.properties

```diff
@@ -37,6 +37,11 @@
 # If true, Falcon skips oozie dryrun while scheduling entities.
 *.falcon.skip.dryrun=false
 
+# Minimum value that the Falcon Oozie Coordinator Builder will use when 
+# configuring the throttle for a coordinator. It is recommended that the
+# minimum be greater than or equal to the Oozie default of 12. 
+*.falcon.oozie.coordinator.throttle.min=12
```

Comment:
This throttling will be applicable only in case of oozie being used as scheduler and not in falcon native scheduler.
Please comment out the same saying that it would work only in case of oozie engine being used as scheduler.